### ISSUE TITLE
Forgified Fabric Api Incompatibility

### ISSUE THREAD

**Issue by CollinToGo** [adde0109/Proxy-Compatible-Forge#40]
Hello i found an incompatibility with Forgified Fabric Api. 
Sadly there is no error but when you join it just doesn't connect you, not really. There is just the "joining world" screen till you get an time out.

Logs **with** FFAPI:
Velocity: https://mclo.gs/buh3I7c
Backend Log: https://mclo.gs/YI7oHlm
Backend Debug Log: https://mclo.gs/98Vusqm

Logs **without** FFAPI:
Velocity: https://mclo.gs/0KbWrib
Backend Log: https://mclo.gs/mgAYp1L

The other NeoForge-Forwarding Mod I know has the same issue with an error, idk maybe it helps..
-> https://github.com/Sonic2423/NeoForwarding/issues/12

Thanks in advance, I hope I provided everything needed... :)

**Comment by p0t4t0sandwich** [comment]
Hoping to have this fixed in the next update

**Comment by luigikartwii** [comment]
Appreciate the work on this! The snapshot you have for this this seems to work on the initial connection. However, it gets stuck on "Joining World" again for any subsequent connections to the server, be it the same person or another. Not sure if you're already aware of this but it seems to be the same as the initial report.

**Comment by p0t4t0sandwich** [comment]
Thanks for confirming that. In testing I only had to remove it the one time, then when reviewing the code and reading through FFAPI some more I was thinking to myself "how the hell was it working after that first connection".
TL;DR, I'll get to fixing that lol
